.github/workflows/builder.yml

```diff
@@ -0,0 +1,297 @@
+      with:
+        path: "${{ github.workspace }}/ubuntu"
+
+    - name: Get information Raspbian
+      id: info_raspbian
+      uses: home-assistant/actions/helpers/info@master
+      with:
+        path: "${{ github.workspace }}/raspbian"
+      
+    - name: Get version
+      id: version
+      uses: home-assistant/actions/helpers/version@master
+      with:
+        type: ${{ env.BUILD_TYPE }}
+  
+  build_alpine:
+    name: Build ${{ matrix.arch }} alpine ${{ matrix.version }} base image
+    needs: init
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: ${{ fromJson(needs.init.outputs.architectures_alpine) }}
+        version: ["3.10", "3.11", "3.12", "3.13"] 
+    steps:
+    - name: Checkout the repository
+      uses: actions/checkout@v2
+
+    - name: Login to DockerHub
+      if: github.event_name == 'release'
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+    - name: Login to GitHub Container Registry
+      if: github.event_name == 'release'
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ secrets.GIT_USER }}
+        password: ${{ secrets.GIT_TOKEN }}
+
+    - name: Set build arguments
+      if: github.event_name != 'release'
+      run: echo "BUILD_ARGS=--test" >> $GITHUB_ENV
+    
+    - name: Set latest tag
+      if: env.ALPINE_LATEST == matrix.version && github.event_name == 'release'
+      run: echo "BUILD_ARGS=--no-latest" >> $GITHUB_ENV
+
+    - name: Build base image
+      uses: home-assistant/builder@2021.03.2
+      with:
+        args: |
+          $BUILD_ARGS \
+          --${{ matrix.arch }} \
+          --target /data/alpine \
+          --with-codenotary "${{ secrets.VCN_USER }}" "${{ secrets.VCN_PASSWORD }}" \
+          --release ${{ needs.init.outputs.release }} \
+          --base ${{ matrix.version }}
+
+  build_debian:
+    name: Build ${{ matrix.arch }} debian ${{ matrix.version }} base image
+    needs: init
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: ${{ fromJson(needs.init.outputs.architectures_debian) }}
+        version: ["stretch", "buster", "bullseye"] 
+    steps:
+    - name: Checkout the repository
+      uses: actions/checkout@v2
+
+    - name: Login to DockerHub
+      if: github.event_name == 'release'
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+    - name: Login to GitHub Container Registry
+      if: github.event_name == 'release'
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ secrets.GIT_USER }}
+        password: ${{ secrets.GIT_TOKEN }}
+
+    - name: Set build arguments
+      if: github.event_name != 'release'
+      run: echo "BUILD_ARGS=--test" >> $GITHUB_ENV
+    
+    - name: Set latest tag
```

Comment:
Shouldn't this be `env.DEBIAN_LATEST != matrix.version`, now it will use `latest` for everything else?

Reply:
Address with 5f32285